code/controllers/subsystem/ticker.dm

```diff
@@ -13,6 +13,7 @@ var/datum/subsystem/ticker/ticker
 
 	var/hide_mode = 0
 	var/datum/game_mode/mode = null
+	var/datum/persistence/persist = null
```

Comment:
Why does it have to be on the ticker? Doesn't seem like a good place for it. Ideally I think it would be easier and better to have a global variable to access which will allow you to access any data from it. Then there would no need to have an infinite loop in New() too.

So for example:

```
var/datum/persistance/game_persistance = new /datum/persistance ()

// later in the code

var/last_singulo_release = game_persistance.get_persistance(LAST_SINGULO_RELEASE)
```